src/app/calculator/process-cooling/cooling-tower/cooling-tower-form/cooling-tower-form.component.html

```diff
@@ -0,0 +1,248 @@
+        <a *ngIf="isBaseline" class="click-link pull-right add-fixture text-danger small pl-2" (click)="removeCase(index)">+Remove Case</a>
+      </label>
+      <label *ngIf="isEditingName" class="group-label w-100">
+        <span class="fa fa-save pr-2 btn-edit-name" (click)="doneEditingName()"></span>
+        <input type="text" id="equipmentName" formControlName="name" onfocus="this.select();" (input)="calculate()">
+        <span class="alert-danger small ml-2" *ngIf="form.controls.name.invalid && !form.controls.name.pristine">
+          <span *ngIf="form.controls.name.errors.required">Please enter a name.</span>
+        </span>
+      </label>
+  
+      <div class="form-section pt-2" *ngIf="isBaseline">
+        <div class="form-group">
+          <label class="small" for="{{'operationalHours_'+idString}}">Annual Operating Hours</label>
+          <div class="input-group">
+            <span class="input-group-addon op-hour-addon units">
+              <a class="click-link" (click)="openOperatingHoursModal()"><span class="fa fa-calculator"></span></a>
+            </span>
+            <input class="form-control" name="operationalHours" id="{{'operationalHours_'+idString}}"
+              formControlName="operationalHours" type="number" (input)="calculate()" (focus)="focusField('operationalHours')" onfocus="this.select();">
+            <span class="units input-group-addon">hrs/yr</span>
+          </div>
+          <span class="alert-danger pull-right small"
+            *ngIf="form.controls.operationalHours.invalid && !form.controls.operationalHours.pristine">
+            <span *ngIf="form.controls.operationalHours.errors.required">Value required.</span>
+            <span *ngIf="form.controls.operationalHours.errors.max">Value can't be greater than
+              {{form.controls.operationalHours.errors.max.max}}.</span>
+            <span *ngIf="form.controls.operationalHours.errors.min">Value can't be less than
+              {{form.controls.operationalHours.errors.min.min}}.</span>
+          </span>
+          </div>
+          
+          <div class="form-group">
+            <label class="small" for="{{'flowRate_'+idString}}">Water Flow Rate</label>
+            <div class="input-group">
+              <input class="form-control" type="number" name="flowRate" id="{{'flowRate_'+idString}}" formControlName="flowRate"
+                (input)="calculate()" (focus)="focusField('flowRate')">
+              <span class="units input-group-addon" *ngIf="settings.unitsOfMeasure == 'Metric'">m<sup>3</sup>/s</span>
+              <span class="units input-group-addon" *ngIf="settings.unitsOfMeasure != 'Metric'">gpm</span>
+            </div>
+            <span class="alert-danger pull-right small"
+              *ngIf="form.controls.flowRate.invalid && !form.controls.flowRate.pristine">
+              <span *ngIf="form.get('flowRate').hasError('greaterThan')">Value must be greater than 0.</span>
```

Comment:
`form.controls.flowRate.errors.greaterThan == 0` will work. 

*ngIf="form.controls.flowRate.errors.greaterThan" didn't work because the value of the error gets set to what it needs to be greaterThan. In this case 0, so it ends up *ngIf="0" which is considered false.